### ISSUE TITLE
Upgrade ExtJS 4.2 to 5.1.1

### ISSUE THREAD

**Issue by daniel-jettka** [Edirom/Edirom-Online-Frontend#17]
Branch: [feature/ExtJS-5.1.1](https://github.com/Edirom/Edirom-Online/tree/feature/ExtJS-5.1.1)

ExtJS 4.2.0 documentation: https://cdn.sencha.com/ext/gpl/4.2.1/docs/#!/api
ExtJS 5.1.1 documenation: https://docs.sencha.com/extjs/5.1.1/index.html

- [x] fix upgrade adviser error messages
- [x] update ExtJS sources to 5.1.1
- [ ] build and deploy and debug

**Comment by bwbohl** [comment]
did you successfully build the app?

**Comment by daniel-jettka** [comment]
> did you successfully build the app?

No, I closed this for maintenance reasons. The parent issue https://github.com/Edirom/Edirom-Online-Frontend/issues/18 with a hint to this one remains open until we get rid of ExtJS

**Comment by bwbohl** [comment]
I see, I successfully upgraded the code to ExtJS 5.1.1 but migrating to the app.json (modernizing the app) resulted in diverse SASS compilation problems, because the compiler changed and it had conflicts with the eoTheme syntax. Everything else went nicely…
